Gemfile

```diff
@@ -1,4 +1,6 @@
 source ENV['GEM_SOURCE'] || 'https://rubygems.org'
+# attempt to resolve gem dependencies by ruby version
+ruby RUBY_VERSION
```

Comment:
yeah, a clean bundler install here _helpfully_ seems to install latest bundler (1.15.0).  but i've also tried it pinned in the Gemfile to 1.13.1 and some 1.14 version.

thanks for the input @ferventcoder!

but maybe it was doing something funny and grabbing a system bundler rather than the rvm installed one :man_shrugging:  
i often times confirmed the bundler version with `bundle --version`
it definitely doesn't quite work the way i'd want as i tried it. 

i'll try a system update or just wipe the local gem cache.